spec.html

```diff
@@ -31084,7 +30948,7 @@
             1. Assert: _items_ is not an Iterable so assume it is an array-like object.
             1. Let _arrayLike_ be ? ToObject(_items_).
             1. Let _len_ be ? ToLength(? Get(_arrayLike_, `"length"`)).
-            1. Let _targetObj_ be ? AllocateTypedArray(_C_, _len_).
+            1. Let _targetObj_ be ? Construct(_C_, «_len_»).
```

Comment:
AllocateTypedArray ensures a new TypedArray instance is created, this is not the case with Construct.